styles/prosilver/template/quickedit.js

```diff
@@ -111,15 +114,35 @@ phpbb.QuickeditAjaxifyEditButtons = function() {
 			refresh: false,
 			callback: fn
 		});
+
+		// Close dropdown in responsive design
+		$this.filter(function() {
+			return !!$(this).closest('.responsive-menu').length;
+		}).click(function() {
+			var $container = $(this).parents('.dropdown-container'),
+				$trigger = $container.find('.dropdown-trigger:first'),
+				data;
+
+			if (!$trigger.length) {
+				data = $container.attr('data-dropdown-trigger');
+				$trigger = data ? $container.children(data) : $container.children('a:first');
+			}
+			$trigger.click();
+		});
 	});
 };
 
-$(document).ready(function() {
+$(window).on('load', function() {
```

Comment:
Because the responsive menu is not formed by the time when document.ready event is fired.
I've come into this solution when doing a similar feature in QuickReply.
Just needs testing to see the difference.